### PR TITLE
Update hardware-transcoding.md

### PR DESCRIPTION
**Pull request by NeZios**
Adding tutorial on how to add QuickSync on Immich dockerfile on Unraid

### CODE REVIEW

File: docs/docs/features/hardware-transcoding.md [hunk 1]
```diff
@@ -100,6 +100,20 @@ immich-microservices:
 
 Once this is done, you can continue to step 3 of "Basic Setup".
 
+#### Immich Docker for Intel QuickSync - Unraid Setup
```
Comment (bo0tzz) [inline docs/docs/features/hardware-transcoding.md:103]:
```suggestion
#### Intel QuickSync
```

**Review by bo0tzz — DISMISSED** [review]
This should be moved down under the `All-In-One - Unraid Setup` header

**Comment by benmccann** [comment]
Looks like you need to run `npm run format`

File: docs/docs/features/hardware-transcoding.md (hunk 1 above)
Comment (mertalev) [reply docs/docs/features/hardware-transcoding.md:103]: It should just be `QSV` to be consistent with the other headers.

File: docs/docs/features/hardware-transcoding.md [hunk 2]
```diff
@@ -100,6 +100,20 @@ immich-microservices:
 
 Once this is done, you can continue to step 3 of "Basic Setup".
 
+#### Immich Docker for Intel QuickSync - Unraid Setup 
+
+1. Unraid > Docker > (Stop) Immich container > Edit
+2. Scroll down and select "Add another path..etc..device"
```
Comment (mertalev) [inline docs/docs/features/hardware-transcoding.md:106]:
```suggestion
2. Scroll down and select `Add another Path, Port, Variable, Label or Device`
```

File: docs/docs/features/hardware-transcoding.md [hunk 3]
```diff
@@ -100,6 +100,20 @@ immich-microservices:
...
+3. In the popup drop down menu select Device
+
+    Name - dev/dri (you are free to put whatever you want) 
+    Value - /dev/dri/
+
```
Comment (mertalev) [inline docs/docs/features/hardware-transcoding.md:111]:
```suggestion
3. In the drop-down menu, select `Device` and an entry with any name and the value `/dev/dri`.
```

File: docs/docs/features/hardware-transcoding.md [hunk 4]
```diff
@@ -100,6 +100,20 @@ immich-microservices:
...
+4. In the Immich web interface : Administration > Settings (left side panel) > Video Transcoding > Hardware Acceleration
+5.  Select - Acceleration Api (QuickSync) then in the "PREFERRED HARDWARE DEVICE FOR TRANSCODING" field you can leave that auto
+
+
```
Comment (mertalev) [inline docs/docs/features/hardware-transcoding.md:115]:
```suggestion
4. Continue to step 4 of "Basic Setup".
```

**Review by mertalev — APPROVED** [review]

**Pull request merged**